src/parser/jobs/sam/changelog.tsx

```diff
@@ -7,6 +7,11 @@ export const changelog = [
 	// 	Changes: () => <>The changes you made</>,
 	// 	contrubutors: [CONTRIBUTORS.YOU],
 	// },
+	{
+		date: new Date('2022-4-18'),
+		Changes: () => <> Removed Kaiten module for parses from patch 6.1+ as it was removed in patch 6.1</>,
```

Comment:
Just killing the space but it could read better.
```suggestion
		date: new Date('2022-04-18'),
		Changes: () => <>Removed Kaiten analysis for the patch 6.1 job update</>,
```